integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb

```diff
@@ -63,7 +63,7 @@
    },
    "outputs": [],
    "source": [
-    "! pip install pytorch-lightning==1.0.0 neptune-client==0.4.123 torch==1.6.0 torchvision==0.7.0"
+    "! pip install pytorch-lightning==1.0.0 neptune-client==0.4.123 torch==1.6.0 torchvision==0.7.0 -f https://download.pytorch.org/whl/torch_stable.html"
```

Comment:
@PiotrJander what does -f  mean here?
Should we still keep torch==1.6.0 in there?

Reply:
Apparently the canonical pip command for installing torch is here https://pytorch.org/get-started/locally/ and for Windows it includes the `-f https://download.pytorch.org/whl/torch_stable.html` option. It doesn't seem to hurt on other platforms.